js/templates/listingCard.html

```diff
@@ -2,15 +2,17 @@
 <div class="gridViewContent">
   <div class="listingImage" style="background-image: url(<%= ob.getServerUrl(`ipfs/${ob.isHiRez() ? ob.thumbnail.medium : ob.thumbnail.small}`) %>), url('../imgs/defaultItem.png')">
       <% if (ob.ownListing) { %>
-        <div class="editOverlay overlayPanel">
-          <div class="overlayPanelInner clrS"></div>
-          <a class='btn clrP clrBr js-edit'><span class="ion-edit"></span></a>
-          <a class='btn clrP clrBr js-delete'><span class="ion-trash-b"></span></a>
+        <div class="editOverlay overlayPanel pad">
+          <div class="overlayPanelInner"></div>
+          <div class="overlayPanelButtons">
```

Comment:
@jjeffryes I needed to add overlayPanelButtons wrapper in order to set a z-index so that the edit/delete buttons display over overlayPanelInner. Without the z-index, the buttons aren't clickable.